lyricsheets/effect/default_effect.py

```diff
@@ -51,7 +51,7 @@ def get_enter_transition_tag(
     resultTag: Sequence[pyass.Tag] = [pyass.AlphaTag(0x00)],
 ) -> pyass.Tag:
     return pyass.TransformTag(
-        start=kChar.fadeOffset, end=switchDuration + kChar.fadeOffset, to=resultTag
+        start=kChar.fadeOffset, end=switchDuration + kChar.fadeOffset, to=list(resultTag)
```

Comment:
Is the explicit conversion necessary when the `TransformTag` constructor already converts it to `Tags`?

Reply:
It isn't but the type checker complains to me since the signature of the TransformTag takes a `Tags` (which is derived from `List`).

It should probably be a `Sequence[Tag]` instead in the signature but that causes some other problems too so I just did this to stop the type checker from screaming at me